spec/commands/asset/delete_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'cache/asset'
+require 'filesystem'
+require 'commands'
+require 'utils'
+
+RSpec.describe Metalware::Commands::Asset::Delete do
+
+  it 'errors if the asset doesnt exist' do
+    expect do
+      Metalware::Utils.run_command(described_class,
+                                   'missing-type',
+                                   'name',
```

Comment:
The `delete` command should only accept a single input. However this spec contains two: `'missing-type', 'name'`. Thus it isn't clear why this spec if failing.

Also a bit further down a `run_command` method has been defined. This spec is currently duplicating that code. This should be condensed down.